### DISCUSSION TITLE
Cancel password authentication

### DISCUSSION THREAD

**Discussion by Pythoneryeah** [coder/code-server#6402] [Q&A] [answered]
![image](https://github.com/coder/code-server/assets/37642828/fe3e66d2-6383-4043-b45d-a795d2f33656)
Hello friends, can the password check in the picture be removed? Have a good suggestion, modify the source code or configuration.

**Comment by lg-nineoneninethreenine** [comment]
Well, I think the password check can be replaced with Pomerium? Other than that, maybe just put `password: none` on `~/.config/code-server/config.yml`. It might be the latter in most cases, but if you were to expose the port, using Pomerium is advantageous, due to OAuth. I think extra data is in the documentation. https://coder.com/docs/code-server/

**Reply by code-asher** [reply, marked as answer]
`auth: none` I believe (or `--auth none` as a flag).
